src/core/output-layout.cpp

```diff
@@ -546,7 +547,13 @@ struct output_layout_output_t
 
         /* It doesn't make sense to transfer to another output if we're
          * going to shut down the compositor */
-        transfer_views(wo, shutdown ? nullptr : get_core().seat->get_active_output());
+        wf::output_t *new_output = shutdown ? nullptr : get_core().seat->get_active_output();
+        transfer_views(wo, new_output);
+        if (new_output &&
```

Comment:
Ehh I'm not quite sure about this particular change. Is this really what users expect?